ember-cli-build.js

```diff
@@ -3,21 +3,22 @@
 const EmberAddon = require('ember-cli/lib/broccoli/ember-addon');
 
 module.exports = function (defaults) {
-  let app = new EmberAddon(defaults, {
-    autoImport: {
-      exclude: ['qunit'],
-    },
-  });
-
-  app.import('node_modules/qunit/qunit/qunit.js', {
-    type: 'test',
-  });
+  let app = new EmberAddon(defaults);
 
   app.import('node_modules/qunit/qunit/qunit.css', {
     type: 'test',
   });
 
-  app.import('vendor/shims/qunit.js', { type: 'test' });
-
-  return app.toTree();
+  try {
```

Comment:
This is verbose, because this library cannot yet drop node 10 and instead requires `@embroider/test-support` be provided via the ember-try-scenario configuration